src/tests/quantuminspire/qiskit/test_backend_qx.py

```diff
@@ -197,6 +199,67 @@ def test_get_experiment_results_returns_correct_value(self):
         self.assertEqual(experiment_result.name, 'circuit0')
         self.assertEqual(experiment_result.shots, number_of_shots)
 
+    def test_get_experiment_results_returns_correct_value_from_latest_run(self):
+        number_of_shots = 100
+        instructions = [{'name': 'h', 'qubits': [0]},
+                        {'name': 'cx', 'qubits': [0, 1]},
+                        {'name': 'measure', 'qubits': [1], 'memory': [1]},
+                        {'name': 'measure', 'qubits': [0], 'memory': [0]}]
+        experiment = self._instructions_to_two_qubit_experiment(instructions)
+        api = Mock()
+        api.get_result_from_job.return_value = {'id': 1, 'histogram': {'1': 0.6, '3': 0.4},
+                                                'execution_time_in_seconds': 2.1, 'number_of_qubits': 2,
+                                                'raw_data_url': 'http://saevar-qutech-nginx/api/results/24/raw-data/'}
+        api.get_raw_data_from_result.return_value = [1] * 60 + [3] * 40
+        job = self._basic_job_dictionary
+        measurements = QuantumInspireBackend._collect_measurements(experiment)
+        user_data = {'name': 'name', 'memory_slots': 2,
+                     'creg_sizes': [['c1', 2]], 'measurements': measurements}
+        job['user_data'] = json.dumps(user_data)
+
+        api.get_job.side_effect = [job]
+        quantuminspire_job = Mock()
+        quantuminspire_job.get_job_identifier.side_effect = [1]
+        qijob = QIJob('backend', '42', api)
+        qijob.add_job(quantuminspire_job)
+
+        simulator = QuantumInspireBackend(api, Mock())
+        experiment_result = simulator.get_experiment_results(qijob, True)[0]
```

Comment:
The meaning of the second argument is not clear. Consider using the explicit parameter name: `only_latest_run=True`

Reply:
Done